Riot/Modules/Room/RoomViewController.m

```diff
@@ -372,6 +374,8 @@ - (void)viewDidLoad
     
     [self.bubblesTableView registerClass:RoomCreationIntroCell.class forCellReuseIdentifier:RoomCreationIntroCell.defaultReuseIdentifier];
     
+    [self.bubblesTableView registerNib:[UINib nibWithNibName:@"RoomTypingBubbleCell" bundle:nil] forCellReuseIdentifier:RoomTypingBubbleCell.cellIdentifier];
```

Comment:
If you inherit `RoomTypingBubbleCell` from `MXKTableViewCell`, you would get `RoomTypingBubbleCell.nib` and `RoomTypingBubbleCell.defaultReuseIdentifier` for free.

Event if we don't do that, we should at least get nib name as `NSStringFromClass`.

Reply:
It's not working in case of swift classes as the class name is composed of the package name and the class name. In this case `NSStringFromClass([RoomTypingBubbleCell class])` returns `Riot.RoomTypingBubbleCell`. Do you want me to convert RoomTypingBubbleCell to objective-C ?

Reply:
Yeah, you're right. Then we can continue with `MXKTableViewCell` solution.